ingestion/src/metadata/ingestion/source/pipeline/dagster/metadata.py

```diff
@@ -214,7 +214,9 @@ def yield_pipeline_status(
             service_name=self.context.pipeline_service,
             pipeline_name=self.context.pipeline,
         )
-        pipeline_entity = self.metadata.get_by_name(entity=Pipeline, fqn=pipeline_fqn)
+        pipeline_entity = self.metadata.get_by_name(
+            entity=Pipeline, fqn=pipeline_fqn, fields=["tasks"]
+        )
```

Comment:
we are only making this API call to get the list of tasks which does not make sense, we should just store the tasks within context and use it as we are doing for other pipeline connectors